examples/network/git2.c

```diff
@@ -57,16 +49,42 @@ int main(int argc, char **argv)
 
 	git_libgit2_init();
 
+	struct args_info args = ARGS_INFO_INIT;
+	const char *git_dir = NULL;
```

Comment:
Declarations after statements are disallowed in C90. Granted, the file here isn't conforming to the C90 standard right now, but that should be no reason to make the situation worse :P